### ISSUE TITLE
config.schema.json: Cannot add elements to array?

### ISSUE THREAD

**Issue by ebaauw** [oznu/homebridge-config-ui-x#289]
I'm trying to add a `config.scheme.json` to my plugins, for homebridge-config-ui-x to generate a user interface for the config.json settings.  The documentation on https://github.com/homebridge/Documentation/blob/master/Schema.md is rather limited, but examining the `config.scheme.json` from homebridge-config-ui-x itself proves quite interesting.  See https://github.com/ebaauw/homebridge-zp and https://github.com/ebaauw/homebridge-ws for my first results.

For the latter plugin, I run into an issue, using the following `config.scheme.json`:
```json
{
  "pluginAlias": "WS",
  "pluginType": "platform",
  "headerDisplay": "Homebridge plugin for virtual weather station",
  "footerDisplay": "Get your OpenWeatherMap API key [here](https://openweathermap.org/price).<br>For a list of valid OpenWeatherMap city names, look [here](https://openweathermap.org/current).",
  "schema": {
    "type": "object",
    "properties": {
      "name": {
        "title": "Name",
	"description": "Plugin name as displayed in the homebridge log.",
        "type": "string",
        "required": true,
        "default": "Weather"
      },
      "apikey": {
        "title": "API key",
	"description": "OpenWeatherMap API key (see below).",
        "type": "string",
	"required": true,
	"pattern": "^[0-9a-f]{32}$"
      },
      "locations": {
        "title": "City Names",
        "type": "array",
	"required": true,
        "items": {
	  "title": "City Name",
	  "description": "OpenWeatherMap city name (see below).",
          "type": "string"
        }
      },
      "timeout": {
        "title": "Timeout",
	"description": "Timeout in seconds.  Default: 15.",
	"type": "integer",
	"minimum": 1,
	"maximum": 60
      }
    }
  }
}
```
The UI seems to render (almost) correctly (I'm puzzled about the _Locations_ label - it looks like a form fieldset?):
![Screenshot 2019-05-06 at 21 01](https://user-images.githubusercontent.com/22179355/57248248-36241e80-7042-11e9-990d-cf243e2d519f.png)

However, the _Add City Name_ button does nothing.  If I add multiple entries in config.json (or even through the _Config_ screen, they show in the UI, and I can delete all but the last entry, but I cannot add entries through the UI.

Not sure if I'm doing something wrong or if this is a bug.  Please advise.

**Comment by oznu** [comment]
@ebaauw - thanks for reporting this. This is resolved in the next release.

**Comment by oznu** [comment]
4.4.0

**Comment by ebaauw** [comment]
Confirmed: fixed in 4.4.0.  Thanks!
